frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -90,6 +91,17 @@ func Dockerfile2LLB(ctx context.Context, dt []byte, opt ConvertOpt) (*llb.State,
 		if st.Name != "" {
 			dispatchStatesByName[strings.ToLower(st.Name)] = ds
 		}
+		if opt.IgnoreCache != nil {
+			if len(opt.IgnoreCache) == 0 {
+				ds.ignoreCache = true
+			} else if st.Name != "" {
+				for _, n := range opt.IgnoreCache {
+					if strings.EqualFold(n, st.Name) {
```

Comment:
Can we support unnamed stages?

Reply:
how?

The name syntax is more making a specific exception, I think it is ok to name a stage if this exception is needed.

We can make it better in follow-up with something like #371 . I'm avoiding doing any Dockerfile changes atm.

Reply:
What was in my mind was `Sprintf("%d", unnamedStageIndex)`, but not needed in this PR.